hack/templates/template.yaml

```diff
@@ -30,53 +30,4 @@ parameters:
   required: true
 metadata:
   name: selectorsyncset-template
-objects:
-- apiVersion: hive.openshift.io/v1alpha1
-  kind: SelectorSyncSet
-  metadata:
-    generation: 1
-    name: ${REPO_NAME}
-  spec:
-    clusterDeploymentSelector:
-      matchLabels:
-        api.openshift.com/managed: "true"
-    resourceApplyMode: sync
-    resources:
-    - apiVersion: operators.coreos.com/v1alpha1
-      kind: CatalogSource
-      metadata:
-        labels:
-          opsrc-datastore: "true"
-          opsrc-provider: redhat
-        name: ${REPO_NAME}-registry
-        namespace: openshift-${REPO_NAME}
-      spec:
-        image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
-        affinity:
-          nodeAffinity:
-            preferredDuringSchedulingIgnoredDuringExecution:
-            - preference:
-                matchExpressions:
-                - key: node-role.kubernetes.io/infra
-                  operator: Exists
-              weight: 1
-        tolerations:
-          - effect: NoSchedule
-            key: node-role.kubernetes.io/infra
-            operator: Exists
-        displayName: ${REPO_NAME}
-        icon:
-          base64data: ""
-          mediatype: ""
-        publisher: Red Hat
-        sourceType: grpc
-    - apiVersion: operators.coreos.com/v1alpha1
-      kind: Subscription
-      metadata:
-        name: ${REPO_NAME}
-        namespace: openshift-${REPO_NAME}
-      spec:
-        channel: ${CHANNEL}
-        name: ${REPO_NAME}
-        source: ${REPO_NAME}-registry
-        sourceNamespace: openshift-${REPO_NAME}
+objects: []
```

Comment:
Is this right? It doesn't look right.

Reply:
Script takes this and populates with child objects, which are SSS in our use case.  It's an array and we have nothing to start with, it is moved to separate yaml in deploy now (CatalogSource and Subscription).  To make sure python uses the right data type, `[]` so yes it's right.